### PR TITLE
Window open close dates

### PR DESCRIPTION
**Pull request by padv2010**
## Trello Card Link

https://trello.com/c/aEqooIHr/129-service-settings-add-a-toggle-to-open-close-applications-service
https://trello.com/c/KBaRff09/89-application-window-closed-page

## Description

Enables/disables the service based on two attributes that are configured via the Settings page on the
admin page. 

image

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -20,3 +20,15 @@
...
+if Rails.env.development?
+  AppSettings.create!(
+    service_start_date: 1.day.ago,
+    service_end_date: 1.year.from_now,
+  )
+
+  AppSettings.current.update!(
+    service_start_date: Time.zone.yesterday,
+    service_end_date: Time.zone.today + 1.year,
+  )
```
Comment (raul-gracia) [inline db/seeds.rb:33]: Why do we create the settings but then updated it again with new dates? 
Shouldn't we just create it first with the right data?

**Review by raul-gracia — APPROVED** [review]

File: app/views/pages/closed.html.erb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <p class="govuk-body">
+      The IRP application window is currently closed.
+    </p>
+    <p class="govuk-body">
+      The next application window opens: (insert the next window opening)
```
[the remark this answers is not in the document]
Comment (padv2010) [reply app/views/pages/closed.html.erb:15]: Yes, very much likely.

At this time, the information needs to be confirmed by Thor so I am leaving a placeholder. I will ask here about your idea of using the start date on the copy, and ask for what content we should put if the date is empty.

File: db/seeds.rb (hunk 1 above)
Comment (padv2010) [reply db/seeds.rb:33]: The reason for creating it twice is the following:

1. I have added it in the migration so all the environments continue working as always. This way we don't have to connect to each one of them to do the update.
2. I have added it to the `seeds.rb` for new developers setting up their environment with `rails db:setup` or `rails db:schema:load`, or for existing developers replanting the seeds with `rails db:seed:replant`.

Hope this makes sense, if not, please let me know.

Comment (padv2010) [reply db/seeds.rb]: Address in: https://github.com/DFE-Digital/get-a-teacher-relocation-payment/pull/155/commits/e4178efa3b3edc9012b8b3a058cf8da03fab711d

**Pull request merged**